docs/build/multi-chain.md

```diff
@@ -0,0 +1,42 @@
+# Multi-Chain Support
+
+- We support indexing multiple networks into to the same database schema. 
+
+- This is useful if you want to gather data from multiple network together in one place. For example capturing transaction data from seperate network (like kusama and polkadot) or capturing xcm data.
+
+## Notes about this feature:
+- It will create multiple metadata tables which you can query using the chainId of the particular subquery project [see below](/build/multi-chain.html#querying)
+- You need to run multiple subql/node instances for each project you are indexing
+- The flag --multi-chain must be enabled from the start of indexing and for all projects. 
+- All multi-chain projects must point to the same schema using `--db-schema=SCHEMA_NAME`
+
+::: info Note
+This feature is not compatible with Historical State and it will be disabled if --multi-chain is used. 
+:::
+
+## Example Project
+
+The repo for the example can be found [here](https://github.com/subquery/multi-networks-transfers)
+
+- This repository contains multiple manifest files which must be indexed with a seperate nodes for now.
+- A modified docker-compose.yaml file has been included two subql/node images, one for each network being indexed.
+- You will notice that that each image maps to a seperate project.yaml file.
```

Comment:
How are IDs set when saving data, do you suggest that people append the chainID to the entity ID when indexing and saving data?

Reply:
If they want to filter by chain they should create a seperate column which they can filter against. I'll include an example.

Reply:
updated